docs/introduction.md

```diff
@@ -0,0 +1,38 @@
+---
+# The default id is the same as the one being defined below. so not needed
+title: Overview
+---
+
+## What is Clarinet?
+
+[Clarinet](https://www.hiro.so/clarinet) provides a CLI package with a [clarity](https://clarity-lang.org/) runtime, an REPL, and a testing harness. Clarinet includes a Javascript library, testing environment, and a browser-based Sandbox. With Clarinet, you can rigorously iterate on your smart contracts locally before moving into production.
+
+Clarinet consists of two components:
+
+- Clarity REPL (Read, Evaluate, Print, Loop)
+- Testing harness.
+
+When the above components are used together, you can rapidly develop and test a Clarity smart contract, with the need to deploy the contract to a local devnet or testnet environments.
```

Comment:
I think these two bullet points are too limited. Should we just let the above paragraph cover everything?